### PR TITLE
feat: new rest methods

### PR DESCRIPTION
**Pull request by DjDeveloperr**
This PR adds some missing methods to interact with REST API.

### CODE REVIEW

**Review by AkiaCode — APPROVED** [review]
lgtm

File: src/models/client.ts [hunk 1]
```diff
@@ -211,6 +213,18 @@ export class Client extends EventEmitter {
     return new Application(this, app)
   }
 
+  /** Fetch an Invite */
+  async fetch(id: string): Promise<Invite> {
```
Comment (Helloyunho) [inline src/models/client.ts:217]: naming it as just `fetch` can confuse with other ones

**Review by Helloyunho — CHANGES_REQUESTED** [review]

File: src/models/client.ts (hunk 1 above)
Comment (DjDeveloperr) [reply src/models/client.ts]: Changed to `fetchInvite`. I was going to do it before, but forgot.

**Review by AkiaCode — APPROVED** [review]

**Review by Helloyunho — APPROVED** [review]
lgtm

**Pull request merged**